zebra/subdir.am

```diff
@@ -101,6 +101,8 @@ zebra_zebra_SOURCES = \
 	zebra/zebra_netns_notify.c \
 	zebra/table_manager.c \
 	zebra/zapi_msg.c \
+	zebra/zebra_nb.c \
+	zebra/zebra_nb_rpcs.c \
```

Comment:
This line needs to be added in the same commit this file was added (to avoid git-rebase issues).